### PR TITLE
Added Translations for Punjabi Language

### PR DESCRIPTION
**Pull request by shivamsawlani02**
@chirag-jn 
I have added Punjabi Language. Please review these changes.

#548

### CODE REVIEW

File: src/main/res/values-pa/strings.xml [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+    <string name="message_error_account_exists">ਖਾਤਾ ਪਹਿਲਾਂ ਹੀ ਮੌਜੂਦ ਹੈ</string>
+    <string name="message_error_search_application">ਸਾਨੂੰ ਇਸ ਫਾਈਲ ਲਈ ਕੋਈ ਸਮਰਥਿਤ ਐਪਸ ਨਹੀਂ ਮਿਲੇ ਹਨ</string>
+    <string name="forgot_password"> <a href="https://www.amahi.org/forgot">ਆਪਣਾ ਪਾਸਵਰਡ ਭੁੱਲ ਗਏ?</a></string>
+    <string name="message_error_authentication">ਉਪਯੋਗਕਰਤਾ ਨਾਮ ਜਾਂ ਪਾਸਵਰਡ ਸਹੀ ਨਹੀਂ ਹੈ. ਮੁੜ ਕੋਸ਼ਿਸ ਕਰੋ ਜੀ.</string>
+    <string name="message_error_authentication_connection">ਕੁਨੈਕਸ਼ਨ ਗਲਤੀ. ਆਪਣੀ ਸੈਟਿੰਗ ਦੀ ਜਾਂਚ ਕਰੋ ਅਤੇ ਦੁਬਾਰਾ ਕੋਸ਼ਿਸ਼ ਕਰੋ.</string>
+    <string name="message_error_authentication_field_empty">ਉਪਯੋਗਕਰਤਾ ਨਾਮ ਜਾਂ ਪਾਸਵਰਡ ਖੇਤਰ ਖਾਲੀ ਨਹੀਂ ਛੱਡਿਆ ਜਾ ਸਕਦਾ</string>
+    <string name="message_error_connection">ਕੁਨੈਕਸ਼ਨ ਗਲਤੀ</string>
+    <string name="message_tap_to_retry">ਦੁਬਾਰਾ ਕੋਸ਼ਿਸ਼ ਕਰਨ ਲਈ ਟੈਪ ਕਰੋ</string>
+    <string name="message_error_video">ਵੀਡੀਓ ਚਲਾਉਣ ਦੌਰਾਨ ਗਲਤੀ</string>
+    <string name="message_error_amahi_anywhere_app">Amahi Anywhere ਤੁਹਾਡੇ ਸਰਵਰ ਵਿੱਚ ਐਪ ਪਹੁੰਚਯੋਗ ਨਹੀਂ ਹੈ. ਕਿਰਪਾ ਕਰਕੇ ਇਹ ਸੁਨਿਸ਼ਚਿਤ ਕਰੋ ਕਿ ਤੁਹਾਡੇ ਨੈਟਵਰਕ ਵਿੱਚ Amahi Anywhere ਐਪ ਸਥਾਪਿਤ, ਚੱਲ ਰਹੀ ਅਤੇ ਪਹੁੰਚਯੋਗ ਹੈ.</string>
```
Comment (chirag-jn) [inline src/main/res/values-pa/strings.xml]: English inside a string. We are translating Amahi at other places too. Please correct this.

File: src/main/res/values-pa/strings.xml [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    <string name="message_file_download_complete">ਫਾਈਲ ਸਫਲਤਾਪੂਰਵਕ ਡਾedਨਲੋਡ ਕੀਤੀ ਗਈ</string>
+    <string name="message_offline_file_deleted">ਫਾਈਲ ਸਫਲਤਾਪੂਰਕ ਹਟਾਈ ਗਈ</string>
+    <string name="notification_download_offline_title">ਉਪਲੱਬਧ ਕਰਵਾਉਣਾ ਔਫਲਾਈਨ</string>
+    <string name="notification_offline_download_complete">ਫਾਈਲ ਉਪਲਬਧ ਹੈ ਔਫਲਾਈਨ</string>
+    <string name="notification_offline_download_paused">ਇੰਟਰਨੈੱਟ ਦੀ ਉਡੀਕ ਕਰ ਰਿਹਾ ਹੈ</string>
+    <string name="notification_offline_download_error">ਫਾਈਲ ਉਪਲੱਬਧ ਕਰਨ ਦੌਰਾਨ ਗਲਤੀ ਔਫਲਾਈਨ</string>
+
+    <string name="share_subject">ਮੇਰੇ Amahi ਘਰ ਦੇ ਸਰਵਰ ਦੀ ਜਾਂਚ ਕਰੋ!</string>
+    <string name="share_message">ਮੈਂ ਆਪਣੀਆਂ ਸਾਰੀਆਂ ਫਾਈਲਾਂ ਨੂੰ ਸਟੋਰ ਕਰਨ, ਬੈਕਅਪ ਲੈਣ ਅਤੇ ਸਟ੍ਰੀਮ ਕਰਨ ਲਈ Amahi ਹੋਮ ਸਰਵਰ ਦੀ ਵਰਤੋਂ ਕਰਦਾ ਹਾਂ.\n\nਇਸ ਦੀ ਜਾਂਚ ਕਰੋ!\n\n<a href="https://www.amahi.org/">https://www.amahi.org/</a></string>
+    <string name="title_version_settings">ਐਂਡਰਾਇਡ ਲਈ Amahi</string>
```
Comment (chirag-jn) [inline src/main/res/values-pa/strings.xml]: English inside a string.

**Review by chirag-jn — CHANGES_REQUESTED** [review]

**Comment by chirag-jn** [comment]
Please add copyright to `values-pa/strings.xml`. Refer `values/strings.xml` for the format and text.

**Comment by shivamsawlani02** [comment]
Ok.I will make these changes

**Comment by shivamsawlani02** [comment]
@chirag-jn I have made the required changes. Please review them.

**Comment by shivamsawlani02** [comment]
Hey @chirag-jn ! Can you please review this pull request and tell me changes(if any)?

**Review by Prakhar-Agarwal-byte — COMMENTED** [review]
@shivamsawlani02 Your branch has conflicts with Amahi: beta branch, please resolve them.

**Comment by chirag-jn** [comment]
Please rebase the branch so that the merge conflicts are resolved.

**Comment by shivamsawlani02** [comment]
@chirag-jn  I have resolved the merge conflicts. Please have a look now.

**Comment by chirag-jn** [comment]
@shivamsawlani02 did you try it locally? Build should have failed locally in my opinion.

File: src/main/res/values-te/strings.xml [hunk 3]
```diff
@@ -45,6 +45,8 @@
    <string name="language_fr">ఫ్రెంచ్</string>
    <string name="language_es">స్పానిష్</string>
    <string name="language_bn">బెంగాలీ</string>
+    <string name="language_pa">పంజాబీ</string>
+    <string name="language_te">తెలుగు</string>
     <string name="language_de">జర్మన్</string>
     <string name="language_te">తెలుగు</string>
```
Comment (chirag-jn) [inline src/main/res/values-te/strings.xml]: `language_te` present twice.

**Review by chirag-jn — CHANGES_REQUESTED** [review]

**Comment by shivamsawlani02** [comment]
> @shivamsawlani02 did you try it locally? Build should have failed locally in my opinion.

Yes I did that locally.

File: src/main/res/values-te/strings.xml (hunk 3 above)
Comment (shivamsawlani02) [reply src/main/res/values-te/strings.xml]: Resolved.

**Pull request merged**